online-resources/learn-to-code-resources.md

```diff
@@ -39,10 +40,12 @@
 * [Famous.co](https://famous.co/)
 * [Free Code Camp](https://learn.freecodecamp.org/)
 * [Free online university courses](https://medium.freecodecamp.org/200-universities-just-launched-560-free-online-courses-heres-the-full-list-d9dd13600b04)
+* [Frontend Masters](https://frontendmasters.com/)
```

Comment:
Never seen this before, nice one 👍

Reply:
Yes, it's really good, especially for really understanding JavaScript. Kyle Simpson (You Don't Know JS series) has many videos there 😄